Lib/test/test_site.py

```diff
@@ -49,6 +49,8 @@ def setUpModule():
             raise unittest.SkipTest('unable to create user site directory (%r): %s'
                                     % (site.USER_SITE, exc))
 
+    # sysconfig._CONFIG_VARS is None until the first call to this function
+    sysconfig.get_config_vars()
```

Comment:
I think it would be better to guard the `sysconfig._CONFIG_VARS.clear()` call in tearDown rather than add an unnecessary initialization.

Reply:
Sounds good. Accessing `sysconfig._CONFIG_VARS` directly is an unsupported operation -- so it's the job of whoever does it to make sure that it works in all cases.